src/hooks/useHandleSelectorComponent.tsx

```diff
@@ -19,7 +19,20 @@ export const useHandleSelectorComponent = ({ item, type }: { item: unknown; type
 
     if (type === SelectorType.ASSIGNEE_SELECTOR) {
       store.dispatch(setCreateTaskFields({ targetField: 'assigneeId', value: (item as IAssigneeCombined)?.id }))
-      store.dispatch(setCreateTaskFields({ targetField: 'assigneeType', value: (item as IAssigneeCombined)?.type }))
+      const assigneeType = (item as IAssigneeCombined).type
```

Comment:
why do we need this casting?

Reply:
MUI Autocomplete component expects `options` with type `unknown` and so for the `useHandleSelectorComponent` hook also I have given the type for the `item` as unknown. 
This type casting are needed to cast the type for the `unknown` `item` type.